state/cleanup.go

```diff
@@ -328,8 +331,8 @@ func (st *State) cleanupMachinesForDyingModel(cleanupArgs []bson.Raw) (err error
 			}
 			continue
 		}
-		// TODO (force 2019-04-26) Should this always be ForceDestroy or only when
-		// 'destroy-model --force' is specified?...
+		// TODO (force 2019-04-26) Should this always be ForceDestroy
+		// or only when 'destroy-model --force' is specified?...
```

Comment:
This TODO needs to go - I have put it in before realizing that for dying models we always want to force destroy non-manual machines. It's redundant now.

Reply:
Ok done!